source/js/characters-per-line.js

```diff
@@ -0,0 +1,56 @@
+/* global errorHandling:false */
+
+var FONTCONSTANT;
+
+/***
+ * @summary Sets font constant
+ * @description For Proportional Fonts and Monospaced-Fonts
+ * @description Font constant (german "Dickte") is the minimum distance between two letters of a font
+ * @description Value comes from how much space a letter takes compares to an m-dash, a dash that is as wide as an m (Geviert).
+ */
+var initFontConstant = function() {
+  var fontFamily = document.querySelector("body").style.fontFamily;
+
+  // Guards
+  var guardFontFamily = errorHandling.checkElement(fontFamily, function(){return true;});
+  if (guardFontFamily === false) {return;}
+
+  switch (fontFamily) {
```

Comment:
Please improve indentation here.

Reply:
We would need to improve the linter in this case. It tells me to intent like so.

Reply:
Yeah, that would be good. I like this indentation style for switch/case clauses: https://google.github.io/styleguide/jsguide.html#formatting-switch-statements

Reply:
incoming :)